programming/sr/leds/index.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: LEDs
+---
+
+LEDs
+====
+
+The KCH HAT provides diagnostic information about the state of your robot
+through a collection of preconfigured and user-controllable LEDs. It is part of
+the [Brain Board](/docs/kit/brain_board) assembly, however control of its LEDs
+is provide through a distinct API.
+
+The LEDs on the KCH can be accessed via the `kch` object:
+
+~~~~~ python
+R.kch.something...
+~~~~~
+
+[LEDs](#leds) {#leds}
+---------------------
+
+The KCH HAT has three RGB LEDs: A, B and C. The LEDs default to off, however
+once set they will hold their value even if your code exits. This is potentially
+useful to understand the current state of your code as it runs or the final
+state of the code afterwards.
+
+Each channel of each LED acts independently, so you can either set them separately:
+
```

Comment:
Might be worth adding here that although the value assigned is a float between `0` and `1`, we currently don't do PWM so it jumps.

Reply:
I wasn't planning to document that it was a float; just leave the examples as `int`s.
Alternatively, is there a reason that the API doesn't just take `bool`s?